spec/declaration.dd

```diff
@@ -545,15 +563,22 @@ $(GNAME AliasReassignment):
 
         $(P An alias declaration inside a template can be reassigned a new value.)
 
+        $(SPEC_RUNNABLE_EXAMPLE_COMPILE
         ---
-        template staticMap(alias F, T...)
+        alias AliasSeq(a...) = a;
```

Comment:
```suggestion
        import std.meta : AliasSeq;
```
?

Reply:
Having the declaration inline isn't such a bad thing but best not to encourage people to redeclare it everywhere